osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/common.xsd

```diff
@@ -0,0 +1,788 @@
+<!-- Copyright 2015 Smart Society Services B.V. Licensed under the Apache 
```

Comment:
This XSD file is named `common.xsd`. A while ago a change has been made to ensure more specific names according to a naming convention: https://github.com/OSGP/open-smart-grid-platform/pull/654

I think this name 'common.xsd' goes back to a generic name, which in a way reverts some of the changes made by 654. Can we come up with a different name, preferably following the naming convention?

Reply:
I will change this file name to base-ws-smartmetering.xsd, as discussed